src/main/java/com/apigee/edge/config/rest/RestUtil.java

```diff
@@ -270,6 +270,54 @@ public static HttpResponse getEnvConfig(ServerProfile profile,
         return response;
     }
 
+    public static HttpResponse updateKvmEntriesForEnv(ServerProfile profile,
+                                                      String kvmName,
+                                                      String entryName,
+                                                      String payload)
```

Comment:
RestUtil.java is intended to be generic and reusable. Can you rename, readjust the method as a overloaded method that takes a subresource.

updateEnvConfig(ServerProfile profile, 
                                                String resource,
                                                String resourceId,
                                                String payload, String subResource)

and in the caller the subResource is "entries". This is needed for Org, Env and API scopes.